pidtree_bcc/plugins/sourceipmap.py

```diff
@@ -0,0 +1,60 @@
+import os
+import staticconf
+from functools import partial
+from pidtree_bcc.plugin import BasePlugin
+
+def hosts_loader(filename):
+    return_dict = {}
+    with open(filename, 'r') as mapfile:
+        hostlines = mapfile.readlines()
+    lines = [line.strip() for line in hostlines if not line.startswith('#') and line.strip() != '']
```

Comment:
nit, iterate directly on the file object and use round parenthesis for lazy evaluation

Reply:
sgtm, this is shameless copypasta ;)

Reply:
So the lazy evaluation (from round parenthesis) does weird things to scope where it things the filehandle is closed (at leaast in pytest) so I've left the square parenthesese, but otherwise fixed.